labs/lab-1-html-and-css/mobilefirststyle.css

```diff
@@ -0,0 +1,206 @@
+}
+
+@media (max-width: 600px){
+
+body {
+  background-color: #f9fafc;
+}
+
+p{
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 800;
+  color: #69696a;
+}
+
+nav a {
+  display: block;
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 800;
+  margin: 10px;
+  position: relative;
+}
+
+a:link {
+  color: #69696a;
+  text-decoration: none;
+}
+
+.top-bar {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+}
+
+.top-header {
+  display: inline;
+}
+
+.main-paragraph {
+  border-left: 3px solid #69696a;
+  padding-left: 15px;
+  margin: 20px;
+}
+
+#scroll-down {
+  position: relative;
+  bottom: 0px;
+  margin: 20px;
+  background-color: black;
+  border-radius: 40px;
+  width: 95%;
+  padding: 10px 0px;
+  color: white;
+  font-size: 14px;
+  text-align: center;
+}
+
+.black-button {
+  background-color: black;
+  border-radius: 40px;
+  width: 100px;
+  padding: 10px 0px;
+  color: white;
+  font-size: 14px;
+  text-align: center;
+  width: 90%;
+  margin: auto;
+
+  }
+}
+@media (min-width: 600px) {
+  body{
+    background-color: #f9fafc;
+  }
+
+  h1{
+    font-family: 'Open Sans', sans-serif;
+    font-weight: 800;
+  }
+
+  h3{
+    font-family: 'Open Sans', sans-serif;
+    font-weight: 800;
+  }
+
+  p{
+    font-family: 'Open Sans', sans-serif;
+    font-weight: 800;
+    color: #69696a;
+  }
+
+  nav a {
+    display: inline;
+    font-family: 'Open Sans', sans-serif;
+    font-weight: 800;
+    margin: 10px;
+    position: relative;
+  }
+
+  a:link {
+    color: #69696a;
+    text-decoration: none;
+  }
+
+  .top-bar {
+    display: inline;
+  }
+
+  .top-header {
+    display: inline;
+  }
+
+  .navigation {
+    float: right;
+  }
+
+  .main-content {
+    margin-top: 100px;
+    display: flex;
+    position: relative;
+  }
+
+  .main-header {
+    margin: 20px;
+  }
+
+  .main-paragraph {
+    border-left: 3px solid #69696a;
+    padding-left: 10px;
+    width: 50%;
+    margin: 20px;
+  }
+
+  .circles {
+    position: relative;
+    width: 50%;
+    height: 400px;
+  }
+
+  .circle-red {
+    position: absolute;
+    width: 300px;
+    height: 300px;
+    border: none;
+    border-radius: 50%;
+    background-color: rgba(247, 136, 167, 0.6);
+    z-index: 1
+
+  }
+
+  .circle-orange {
+    width: 200px;
+    height: 200px;
+    position: absolute;
+    border: none;
+    border-radius: 50%;
+    background-color: rgba(247, 193, 162, 0.7);
+    z-index: 3;
+    left: 220px;
+    top: 140px;
+
+  }
+
+  .circle-purple {
+    width: 150px;
+    height: 150px;
+    position: absolute;
+    border: none;
+    border-radius: 50%;
+    background-color: rgba(207, 130, 213, 0.6);
+    z-index: 2;
+    top: 30px;
+    left: 250px;
+
+  }
+
+  .black-button {
+    background-color: black;
+    border-radius: 40px;
+    display: inline;
+    width: 100px;
+    height: 50px;
+    padding: 5px 20px;
+    color: white;
+    font-size: 14px;
+
+  }
+
+  #scroll-down {
+    position: absolute;
```

Comment:
I wouldn't expect to see this positioned this way. You've got the right idea with `margin: auto`, but you can't do that AND `position: absolute`. Maybe nest your margin-y button *inside* a div that's `position: absolute`?